tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -578,10 +578,9 @@ def test_model_monitoring_with_regression(self):
         # ) + len(model_endpoint.spec.label_names)
 
         # Validate monitoring feature set URI
-        m_fs = mlrun.feature_store.get_feature_set(
-            f"store://feature-sets/{self.project}/monitoring-serving-diabetes_model-latest:latest"
```

Comment:
I think its worth checking the structure of the URI and not simply check its not an empty string or `None`.
You should do both asserts in my opinion.